ivy/functional/frontends/torch/other_operations.py

```diff
@@ -0,0 +1,5 @@
+import ivy
+
+
+def clone(input, *, memory_format=torch.preserve_format):
```

Comment:
This function is not on ivy's functional API

Reply:
Thanks,

Am I on the right path cause I am following the documentation https://pytorch.org/docs/stable/generated/torch.clone.html
and the video https://www.youtube.com/watch?v=1crW78zKxSc&list=PLwNuX3xB_tv-wTpVDMSJr7XW6IP_qZH0t&index=5&ab_channel=Ivy . Please let me know if am following the correct path.

Thanks again,